src/components/validations/validation-types.constant.js

```diff
@@ -0,0 +1,5 @@
+export default {
```

Comment:
These should probably either be actual constants, or the file should be renamed.